src/services/handlers/searchApi.ts

```diff
@@ -17,17 +18,18 @@ const getSearchResults = async (searchFieldsObject: ISearchPayload): Promise<ISe
     }
     /* eslint-disable  @typescript-eslint/no-explicit-any */
   } catch (error: any) {
+    console.log(error);
```

Comment:
please remove this console.log if not required

Reply:
Yes, it is not required.